docker/candlepin-base/cp-test

```diff
@@ -231,4 +241,12 @@ if [ "$RSPEC" == "1" ]; then
     buildr "${BUILDR_ARGS}" > /tmp/teepipe 2>&1
 fi
 
+if [ ! -z "$ARBITRARY" ]; then
+    cd "$CP_HOME"
+    echo "Running buildr $ARBITRARY..."
+    tee /var/log/candlepin/buildr.log < /tmp/teepipe &
```

Comment:
One more request that came up during my manual testing:

We should flip the tee declaration and echo here, then output the "Running buldr ..." to the teepipe so it shows up in the log. Right now it doesn't, so the log doesn't show what command was run.